.circleci/src/commands/install_fvm.yml

```diff
@@ -4,20 +4,25 @@ steps:
       command: |
         dart pub global activate fvm
         echo 'export PATH=$HOME/.pub-cache/bin:$PATH' >> $BASH_ENV
+        export PATH="$PATH":"$HOME/.pub-cache/bin"
   - restore_cache:
       keys:
-        - fvm-0-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+        - fvm-0-{{ checksum "frontend/.fvm/fvm_config.json" }}-{{ arch }}
   - run:
       name: Install Flutter
       command: fvm install
+      working_directory: ~/project/frontend/
   - save_cache:
-      key: fvm-0-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+      key: fvm-0-{{ checksum "frontend/.fvm/fvm_config.json" }}-{{ arch }}
       paths:
         - .fvm
         - ~/fvm/
+      working_directory: ~/project/frontend/android
```

Comment:
Why this working directory? Shouldn't it be just ~/project/frontend?

Reply:
Thanks, adjusted it.